_includes/images/color-functions.html

```diff
@@ -1,4 +1,5 @@
 
+{::options parse_block_html="false" /}
```

Comment:
This is needed to prevent the Markdown compiler from wrapping the `img` element in a `p`.